addons/transform/org.openhab.transform.feed/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Feed Transformation Service
+Bundle-SymbolicName: org.openhab.transform.feed
```

Comment:
Thanks, I will change it. I guess this partly answers why :-) :

> "The term 'singleton' as used in the OSGI specification does not imply the Singleton design pattern, but rather specifies that the plugin is a singleton bundle. When enabled, the symbolic bundle name can only resolve to a single version of the bundle, as opposed to being capable of resolving to two or more version having the same bundle name."

From https://www.experts-exchange.com/questions/24999145/OSGi-bundle-Singleton.html